### PR TITLE
Add tests for kubectl qos

### PR DESCRIPTION
**Pull request by lixiv**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

add unit tests

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by Octopusjust — COMMENTED** [review]
/ok-to-test

File: staging/src/k8s.io/kubectl/pkg/util/qos/qos_test.go [hunk 1]
```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```
Comment (Octopusjust) [inline staging/src/k8s.io/kubectl/pkg/util/qos/qos_test.go:2]: 2024

File: staging/src/k8s.io/kubectl/pkg/util/qos/qos_test.go [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+				},
+			},
+			want: corev1.PodQOSBestEffort,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetPodQOS(tt.pod); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("GetPodQOS() = %v, want %v", got, tt.want)
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/util/qos/qos_test.go:197]:
```suggestion
				t.Errorf("unexpected PodQOSClass actual %s, expected %s", got, tt.want)
```

**Review by ardaguclu — COMMENTED** [review]
Thank you. One nit comment, other than that looks good
/triage accepted
/priority backlog

**Comment by ardaguclu** [comment]
Also could you please give more details in the section of `What this PR does / why we need it:` to better reflect the PR's outcome?

**Pull request merged**